lib/hsa/hsaRfiMaskOutput.hpp

```diff
@@ -47,37 +45,15 @@ class hsaRfiMaskOutput : public hsaSubframeCommand {
     /// The raw time series data from the FPGAs
     Buffer* _network_buf;
     /// The array of missing sample flags for _network_buf
-    Buffer* _lost_samples_buf;
-    /// Output buffer from the FRB pipeline
     Buffer* _rfi_mask_output_buf;
-    /// Output of the N2 correlation products
-    Buffer* _output_buf;
     /// ID for _network_buf
     int32_t _network_buf_id;
-    /// ID for _lost_samples_buf
-    int32_t _lost_samples_buf_id;
-    /// ID for _output_buf;
-    int32_t _output_buf_id;
     /// ID for _rfi_mask_output_buf
     int32_t _rfi_mask_output_buf_id;
     /// ID for _rfi_mask_output_buf_precondition
     int32_t _rfi_mask_output_buf_precondition_id;
     /// ID for _rfi_mask_output_buf_execute
     int32_t _rfi_mask_output_buf_execute_id;
-    // General Config Parameters
-    /// Number of elements (2048 for CHIME or 256 for Pathfinder)
-    uint32_t _num_elements;
-    /// Number of frequencies per GPU (1 for CHIME or 8 for Pathfinder)
-    uint32_t _num_local_freq;
-    /// Total number of frequencies (1024)
-    uint32_t _num_total_freq;
```

Comment:
Also never used?

Reply:
Ah this was used at one point when it supported Pathfinder type configurations.  However we decided to remove that support, but it looks like this wasn't cleaned up.